qp_klp/Protocol.py

```diff
@@ -129,33 +131,39 @@ def convert_raw_to_fastq(self):
 
     def generate_sequence_counts(self):
         config = self.pipeline.get_software_configuration('tell-seq')
+        # filter on corrected.err_barcode_removed
+
+        files_to_count_path = join(self.pipeline.output_path,
+                                   'files_to_count.txt')
+
+        with open(files_to_count_path, 'w') as f:
+            # for raw_counts_r1r2, count corrected.err_barcode_removed files
+            # (TellReadJob final output).
+            for root, dirs, files in walk(self.raw_fastq_files_path):
+                for _file in files:
+                    if 'corrected.err_barcode_removed' in _file:
```

Comment:
Why are these files being examined?

Reply:
yup, I have this marked in my notes. there is a patch for this that expects the integrated results directory instead of Full.